core/src/Pos/Core/Txp/TxWitness.hs

```diff
@@ -28,12 +28,12 @@ type TxWitness = Vector TxInWitness
 
 -- | A witness for a single input.
 data TxInWitness
-    = PkWitness { twKey :: !PublicKey
-                , twSig :: !TxSig }
-    | ScriptWitness { twValidator :: !Script
-                    , twRedeemer  :: !Script }
-    | RedeemWitness { twRedeemKey :: !RedeemPublicKey
-                    , twRedeemSig :: !(RedeemSignature TxSigData) }
+    -- PkWitness twKey twSig
+    = PkWitness PublicKey TxSig
+    -- ScriptWitness twValidator twRedeemer
+    | ScriptWitness Script Script
+    -- RedeemWitness twRedeemKey twRedeemSig
+    | RedeemWitness RedeemPublicKey (RedeemSignature TxSigData)
```

Comment:
Some strictness annotations were lost here - it's recommended to keep those: https://github.com/serokell/serokell-util/blob/master/serokell-style.md#dealing-with-laziness

Reply:
Whoops, thanks @mhuesch